### ISSUE TITLE
Use Editable fields for metadata

### ISSUE THREAD

**Issue by franky47** [franky47/post.francoisbest.com#16]
Especially the Twitter handle, when the user is known (eh: dhh, levelsio) but not present in the crawled data.

**Comment by eckertalex** [comment]
I do not feel there is a value in making the OpenGraph Image or Logo editable. It is too much effort to find and upload the correct image for a new post. What do you think @franky47?

**Comment by franky47** [comment]
Yes, we should be able to update only text fields in the property/value table. Editing the post date makes little sense though, as it's rarely available on the page and not on metadata.

**Comment by eckertalex** [comment]
Do you mean the publication date of the post? If so I agree, and will remove it again from #22

**Comment by franky47** [comment]
Yes.

Would it be possible to use the Editable series of components instead of Inputs to be more discrete ?

Also, the Twitter handle used to be a clickable link, I think this behaviour should remain available, maybe with an icon link next to the handle.

**Comment by eckertalex** [comment]
I can make the text fields an editable field. The twitter input actually has exactly what you suggest in the current PR. I will add a picture.

**Comment by franky47** [comment]
Perfect, sorry I voiced some thoughts without checking the preview, great job!

**Comment by eckertalex** [comment]
Here is a screenshot of how it looks like with a twitter handle.

<img width="558" alt="Screenshot 2021-06-26 at 22 06 46" src="https://user-images.githubusercontent.com/19299600/123524468-d928ce00-d6ca-11eb-9785-a038331442ef.png">

I was unable to convert these fields to chakraUIs Editable fields. Specifically making it look nice with the loading indicator.

**Comment by eckertalex** [comment]
Also the label did not focus the field anymore. There were other smaller issues as well. You're welcome to try add that in my PR.
